src/lib/support/tests/ExtraPwTestMacros.h

```diff
@@ -86,17 +86,17 @@
 #define EXPECT_SUCCESS(expr) EXPECT_EQ((expr), CHIP_NO_ERROR)
 
 /**
- *  @def EXPECT_SUCCESS(expr)
```

Comment:
You're going to be changing [*a lot of places*, over 93 files](https://github.com/search?q=repo%3Aproject-chip%2Fconnectedhomeip+%22+EXPECT_SUCCESS%28%22&type=code) if you do this, I don't know if it's worthwhile; also, it's not an assert, it's a pw unit test expectation; see https://github.com/project-chip/connectedhomeip/pull/41796#discussion_r2495576124